custom/icds_reports/utils/aggregation_helpers/distributed/inactive_awws.py

```diff
@@ -16,12 +16,25 @@ def __init__(self, last_sync):
         self.last_sync = last_sync
 
     def aggregate(self, cursor):
+        delete_extra_record_query = self.drop_table_query()
```

Comment:
This isn't calling the method you just introduced

Reply:
updated.